README.md

```diff
@@ -109,6 +109,7 @@ Once you've started an `sbt` session you can run one of the core commands:
   - `compile` compiles all sub-projects (library, reflect, compiler, scaladoc, etc)
   - `scala` / `scalac` run the REPL / compiler directly from sbt (accept options /
     arguments)
+  - `enableOptimizer` reloads the build with the Scala optimizer enabler. Our releases are build in this way. Enable this when working on compiler performance improvements.
```

Comment:
Maybe mention `setupPublishCore`, as it also sets a nicer version number, and makes `publishLocal` faster by skipping docs?

Reply:
I always have to scrounge my memory for `setupPublishCore`. OK I see now that it autocompletes, so I would mention that completion works. (Though `spc` does not autocomplete, which I mention because I am spoiled.)

`s/optimizer enabler/optimiser enabled`. `s/z` is optional, I guess. Maybe there really is an `optimizer enabler`.

`s/are build/are built`.

"completion works", but I don't understand why completions of "set" include "every". I'll read the sbt docs someday.

Reply:
Surely `enableOptimizer` is the optimiser enabler. You could also say "Disable this as a compiler performance improvement", which it certainly isn't, at least when not bootstrapping.